src/prefect/telemetry/run_telemetry.py

```diff
@@ -1,8 +1,9 @@
 import time
 from dataclasses import dataclass, field
-from typing import TYPE_CHECKING, Any, Dict, Optional, Union
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Union
```

Comment:
as a part of https://github.com/PrefectHQ/prefect/issues/16292 we've been trying to move `Dict[str, Any]` to `dict[str, Any]` and same for `tuple`, `list` `set` etc
```suggestion
from typing import TYPE_CHECKING, Any, Optional, Union
```